static/app/views/alerts/rules/issue/previewTable.tsx

```diff
@@ -98,7 +98,7 @@ function PreviewTable({
   return (
     <IssuesReplayCountProvider groupIds={previewGroups || []}>
       <Panel>
-        <GroupListHeader withChart={false} showLastTriggered />
```

Comment:
Did we need to add the `withColumns={["lastTriggered"]}`

Reply:
would have to list all the columns but yeah we'd still want lastTriggered here

Reply:
Yeah, nice catch!